internal/pkg/cli/pipeline_list.go

```diff
@@ -35,11 +36,12 @@ type listPipelineVars struct {
 
 type listPipelineOpts struct {
 	listPipelineVars
-	pipelineSvc pipelineGetter
-	prompt      prompter
-	sel         configSelector
-	store       store
-	w           io.Writer
+	pipelineInfoGetter pipelineGetter
+	deployedPipelineLister
```

Comment:
do you want to add a field here instead of embedding the interface?

```suggestion
	codepipeline pipelineGetter
	pipelineLister deployedPipelineLister
```